includes/degree-import-functions.php

```diff
@@ -148,6 +148,17 @@ function mainsite_degree_format_post_data( $meta, $program ) {
 		}
 	}
 
+	$meta['highlights_imported'] = array();
+	if ( isset( $program->highlights) ) {
+		foreach ( $program->highlights as $index => $highlight) {
+			$meta['highlights_imported'][] = array (
+				'highlight_imported_icon_name' => $highlight->icon_class,
+				'highlight_imported_copy' => $highlight->description
+			);
+		}
+		update_field('highlights_imported', $meta['highlights_imported']);
```

Comment:
The degree importer will run `update_field` on every item within the `$meta` array. This line should be removed, as the update function is being called twice with the addition of this one.

Reply:
The update_field function was removed.